addons/mail/static/src/messaging.js

```diff
@@ -263,6 +273,15 @@ export class Messaging {
                         this.createMessage(body, message, thread);
                     }
                     break;
+                case "mail.record/insert":
+                    {
+                        const partners_in_payload = notif.payload.Partner;
+                        for (const partner in partners_in_payload) {
```

Comment:
the `in` iterable is `partnerId`, not `partner`. Renaming accordingly will improve readability, and even simplifying code (`partners_in_payload[partner].id` is just `partnerId`)